twisted/internet/defer.py

```diff
@@ -505,7 +505,7 @@ def _startRunCallbacks(self, result):
     def _continuation(self):
         """
         Build a tuple of callback and errback with L{_CONTINUE} to be used by
-        L{_addContinue} and L{_removeContinue} on another Deferred.
+        L{_runCallbacks}.
```

Comment:
I think that referencing the name of the method who are using this in the docstring is bad and the docstring should be rephrased to just inform what it does and not who is calling it.